src/test/groovy/org/cassalog/core/CassalogTest.groovy

```diff
@@ -104,6 +105,28 @@ class CassalogTest extends CassalogBaseTest {
     assertChangeSetEquals(rows[1], new ChangeSet(version: change2Id, author: 'admin', description: 'test'))
   }
 
+  @Test
```

Comment:
I noticed the tests leave a lot of keyspaces in cassandra, should I change them so it leaves cassandra in the same exact status it was?